### ISSUE TITLE
Undefined symbols for architecture armv7:   "_OBJC_CLASS_$_UnityAppController"

### ISSUE THREAD

**Issue by nicholaswhittle** [asmadsen/react-native-unity-view#53]
Hi guys

Does anyone know why i'm getting this error when trying to build:

```
Undefined symbols for architecture armv7:
  "_OBJC_CLASS_$_UnityAppController", referenced from:
      objc-class-ref in libreact-native-unity-view.a(UnityUtils.o)
      objc-class-ref in libreact-native-unity-view.a(RNUnityViewManager.o)
ld: symbol(s) not found for architecture armv7
```

I recently updated unity version, i can build and run just fine to physical device. But when trying to build to archive im presented with the above error?

**Comment by harishsn** [comment]
any solution ?

**Comment by nicholaswhittle** [comment]
Yeah so it appears 2020 version of unity does not build the unityframework for armv7 anymore. 
So setting the architecture in Xcode to arm64 only solved the issue. 

Running into issues on android now though!

**Comment by daniel-keen** [comment]
Everything is set to arm64. Unfortunately that didn't help. Is there any other possible solution to the issue?

**Comment by SIYABARSHADFUSIONTECH** [comment]
These two solutions worked for me
**First one** 
 have solved this by selecting armv7 and arm64 both on buildsetting it worked for me.
**Second One**
Just Downgrade the Unity version because after 2020.1 unity does not support armv7 by default so choose 2019 or 18 but make sure to back-up your Unity project
